triggering_edge_multi_low_level_test.go

```diff
@@ -33,23 +33,23 @@ func TestEdgeMultiParts1(t *testing.T) {
 	assert.Equal(t, FrameIndex(12), s.v, "EMTState should have u==v, to indicated that u has has been recordized")
 
 	// two records far enough apart to trigger both
-	raw_A := [21]RawType{0, 0, 0, 0, 10, 20, 0, 0, 10, 20, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
```

Comment:
The underscore was useful to make it easy to copy and paste then select all `_A` and change all to `_B`. This change is ok though.